rpm/maep-qt.spec

```diff
@@ -69,6 +69,7 @@ install -m 644 -p /usr/lib/libfreetype.so.6 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicui18n.so.52 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicuuc.so.52 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicudata.so.52 %{buildroot}/usr/share/%{name}/lib/
+install -m 644 -p /lib/ld-linux.so.? %{buildroot}/usr/share/%{name}/lib/
 
```

Comment:
Ok, I see, you've tried to build for the i486, haven't you ? For the arm target, the ld-linux-arm.so stuff is indeed allowed.

To be short, I think ld-linux.so.2 is not existing for the arm target and thus cannot be installed as your patch is suggesting. I'll fill a PR for the ```rpmvalidation.sh tool```.